mantidproject/mantid
# Core and Diffraction Release Notes 6.0

**Description of work.**

Reformatted release notes for release 6.0 for mantidworkbench, index,
diffraction and framework pages
Images have been included where possible

**To test:**

- Check they have a common language.
- Use common layout, and check the structure matches the [Docs Guidelines](https://developer.mantidproject.org/Standards/DocumentationGuideForDevs.html#how-to-define-titles-sections-etc)
- There are images that appear correctly in the release notes, well formatted, and relate clearly to a particular bullet point, highlighted in **bold**.
- Check links work
- Check for typos (say by pasting all text into MSWord)

Fixes ##30372

*This does not require release notes* because **it is release notes**




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
